src/components/data-processing/CalculationConfigContainer.jsx

```diff
@@ -69,9 +69,11 @@ const CalculationConfigContainer = (props) => {
       {React.cloneElement(children, {
         config,
         plotType,
-        updateSettings: (diff) => {
+        updateSettings: (diff, outstandingChanges = true) => {
           dispatch(updateFilterSettings(filterUuid, diff, sampleId));
-          onFilterSettingsChange();
+          if (outstandingChanges) {
+            onFilterSettingsChange();
+          }
```

Comment:
This code means that there are **setting changes** which does not cause **outstanding changes**, which is confusing. I see that you do this so that you can change the settings without causing the popup to showup. Is there a better way to do this?